specification/redisenterprise/resource-manager/Microsoft.Cache/stable/2024-10-01/redisenterprise.json

```diff
@@ -0,0 +1,2145 @@
+{
+  "swagger": "2.0",
```

Comment:
>swagger [](http://example.com/codeflow?start=3&length=7)

There are few more breaking changes as redundancy mode and other values are not in the latest stable one. Can you confirm that as well once.

Reply:
The changes which are part of the preview but not in stable are not considered as breaking changes right.
this stable version is based on the previous stable version and added some more which are ready for GA. 
We already got the approval for the parameter regex change.

Reply:
Thanks